### ISSUE TITLE
ARC

### ISSUE THREAD

**Issue by andsmi** [0xSina/URLConnection#1]
I added ARC and a Readme to my own branch...is this still the optimal way to handle a connection with blocks without including a big library?

**Comment by 0xSina** [comment]
No. In IOS 6 you have + sendAsynchronousRequest:queue:completionHandler: for NSURLConnection.

**Comment by andsmi** [comment]
But for iOS 5 and iPad 1 I still need this, right?

**Comment by 0xSina** [comment]
Yes

**Comment by andsmi** [comment]
According to do the docs that is available in iOS 5 as well.... http://developer.apple.com/library/ios/documentation/Cocoa/Reference/Foundation/Classes/NSURLConnection_Class/Reference/Reference.html#//apple_ref/occ/clm/NSURLConnection/sendAsynchronousRequest:queue:completionHandler:

**Comment by 0xSina** [comment]
Cool, didn't know that.

**Comment by andsmi** [comment]
Would you be willing to put an update at top of the blog post noting that? Your blog post on this was great and came up first when I was looking for this solution-- but it would be nice if it now directed to the new api from ios 5

**Comment by 0xSina** [comment]
Done!
